core/block/editor/profile.go

```diff
@@ -123,6 +130,10 @@ func (p *Profile) StateMigrations() migration.Migrations {
 			Version: 3,
 			Proc:    migrationSetHidden,
 		},
+		//{
```

Comment:
Why did we comment it?

Reply:
I put comment here to show that we can use StateMigration mechanism alternatively.
I will remove comments while we choose more applicable variant

Reply:
We decided to do migration in buildState